src/constants/flows/actions/sendMessage.ts

```diff
@@ -21,8 +24,13 @@ const sendMessage: Action<[Snowflake, string]> = {
           .replace(/\{everyone\}/giu, guild.roles.everyone.toString())
           .replace(/\{score\}/giu, String(scores.get(member.id) ?? 0))
           .replace(/\{content\}/giu, content),
-        allowedMentions: { parse: ["everyone", "users", "roles"]},
-      }).catch();
+        allowedMentions: {
+          users: [...text.matchAll(/<@(\d*)>/gu)] as unknown as string[],
```

Comment:
The `as unknown as string[]` is a little bit of a hack, can anyone think of a better way?

Reply:
use `[...text.match(..) ?? []]` instead

Reply:
also, it still spits out `<@123>` instead of `123` (both .match and .matchAll) so we probably need a .map to only include the digits

Reply:
oh yeah, totally forgot about the match groups